platform/viewer/src/connectedComponents/ConnectedDownloadDialog.js

```diff
@@ -0,0 +1,106 @@
+import { connect } from 'react-redux';
+import { DownloadDialog } from '@ohif/ui';
+import b64toBlob from '../lib/utils/b64toBlob';
+
+const MINIMUM_SIZE = 100;
+const DEFAULT_SIZE = 512;
+
+const mapStateToProps = (state, ownProps) => {
+  const { viewportSpecificData, activeViewportIndex } = state.viewports;
+  const { dom: activeEnabledElement } = viewportSpecificData[activeViewportIndex] || {};
+
+  return {
+    minimumSize: MINIMUM_SIZE,
+    defaultSize: DEFAULT_SIZE,
+    canvasClass: "cornerstone-canvas",
+    onClose: ownProps.toggleDownloadDialog,
+    activeViewport: activeEnabledElement,
+    enableViewport: viewportElement => {
+      if (viewportElement) {
+        cornerstone.enable(viewportElement);
```

Comment:
Is this getting `cornerstone` from some global declaration? Can we import it instead of relying on it being set globally?

Reply:
You are totally right, done. thanks.